<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>353</width>
    <height>190</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>353</width>
    <height>190</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>353</width>
    <height>190</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Waiting</string>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>331</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Wait for the specified condition(s), then delay for ___ seconds before continuing with the sequence.</string>
   </property>
   <property name="wordWrap">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>70</y>
     <width>141</width>
     <height>91</height>
    </rect>
   </property>
   <property name="title">
    <string>Condition(s)</string>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <widget class="QCheckBox" name="check_Temp">
      <property name="text">
       <string>Temperature</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QCheckBox" name="check_Field">
      <property name="text">
       <string>Field</string>
      </property>
     </widget>
    </item>
   </layout>
   <zorder>check_Temp</zorder>
   <zorder>check_Temp</zorder>
   <zorder>check_Field</zorder>
  </widget>
  <widget class="QDialogButtonBox" name="buttonDialog">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>110</y>
     <width>121</width>
     <height>61</height>
    </rect>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="autoFillBackground">
    <bool>true</bool>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
   <property name="centerButtons">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>70</y>
     <width>120</width>
     <height>31</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>Delay (secs):</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QSpinBox" name="spin_delayseconds">
      <property name="maximum">
       <number>3600</number>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
